### PR TITLE
Fixed build

### PR DESCRIPTION
**Pull request by Andre3567**
Had to make a few changes to fix the build process.

### CODE REVIEW

**Review by Andrei-C96 — APPROVED** [review]
LGTM!

File: build.gradle [hunk 1]
```diff
@@ -98,11 +104,11 @@ task sampleLibs(type: Copy, dependsOn: 'sampleTemplate') {
 task sampleProperties (dependsOn: 'sampleTemplate') {
     doLast {
       def version = rootProject.ext.tritonSDKVersion
-      logger.info("Create saple gradle.properties")
+      logger.info("Create sample gradle.properties")
```
Comment (EddieGr) [inline build.gradle:106]: 😆

File: player/src/androidTest/java/com/tritondigital/player/CuePointHistoryAndroidTest.java [hunk 2]
```diff
@@ -45,7 +45,7 @@ public void tearDown() {
     public void canGetLastCuePoint() {
         Runnable r = new Runnable() {
             public void run() {
-                mParser.setMount("S1_FLV_MP3");
```
Comment (EddieGr) [inline player/src/androidTest/java/com/tritondigital/player/CuePointHistoryAndroidTest.java:48]: Is this one not used anymore?

**Review by EddieGr — APPROVED** [review]

File: util/google-services.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{
+  "project_info": {
+    "project_number": "915336943764",
+    "project_id": "triton-android-sdk",
+    "storage_bucket": "triton-android-sdk.appspot.com"
+  },
```
Comment (dral3x) [inline util/google-services.json]: Why adding this to the public repo? This should be something private IMO

**Review by dral3x — CHANGES_REQUESTED** [review]

File: player/src/androidTest/java/com/tritondigital/player/CuePointHistoryAndroidTest.java (hunk 2 above)
Comment (Andre3567) [reply player/src/androidTest/java/com/tritondigital/player/CuePointHistoryAndroidTest.java:48]: Yes the station isn't playing which caused the tests to fail.

File: util/google-services.json (hunk 3 above)
Comment (Andre3567) [reply util/google-services.json]: I agree this should be in here.

**Review by dral3x — APPROVED** [review]
LGTM 👍

**Pull request merged**